src/extension/wps/src/main/java/au/org/emii/wps/GoGoDuckProcess.java

```diff
@@ -69,7 +76,11 @@ public FileRawData execute(
                 throw new ProcessException("fileLimit set to 0 or below, job will not run");
             }
 
-            List<Converter> converters = GoGoDuckUtils.addFilters(filter);
+            List<Converter> converters = new ArrayList<Converter>();
```

Comment:
Probably.  I've switched to using mimetypes instead of filters but haven't followed through everywhere.  I'll revisit.